src/wallet/wallet.cpp

```diff
@@ -1476,7 +1476,18 @@ set<uint256> CWallet::GetConflicts(const uint256& txid) const
         }
     }
 
-    // TODO ORCHARD; see #5593
+    for (int i = 0; i < wtx.GetOrchardBundle().GetNumActions(); i++) {
+        OrchardOutPoint op(i, wtx.GetHash());
```

Comment:
These are backwards:
```suggestion
        OrchardOutPoint op(wtx.GetHash(), i);
```